query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/converters/PostgresFunctionConverterConfig.java

```diff
@@ -50,6 +63,11 @@ public PostgresFunctionConverterConfig(Config config) {
     } else {
       this.distinctCountAggregationFunction = DEFAULT_DISTINCT_COUNT_AGGREGATION_FUNCTION;
     }
+    if (config.hasPath(UNNEST_FUNCTION_CONFIG)) {
```

Comment:
why do we need a config for unnest - it's an internal thing, right??

Reply:
Done. Removed the config part.